Products/ZenUI3/browser/search/resources/zenoss-search.js

```diff
@@ -59,18 +48,50 @@ Ext.onReady(function () {
             idProperty:'url',
             fields:[
                 {name:'url'},
+                {name:'id'},
+                {name:'name'},
                 {name:'content'},
                 {name:'popout'},
-                {name:'category'}
+                {name:'category'},
+                {name:'isSavedResult', type: 'boolean', convert: function(val, record) {
+                    return !!record.get('id');
+                }}
             ]
         });
 
         // Create the Store for the Free Search
         var freeSearchDataStore = Ext.create('Zenoss.NonPaginatedStore', {
-            storeId:freeSearchDataStoreID,
-            directFn:Zenoss.remote.SearchRouter.getLiveResults,
+            storeId: freeSearchDataStoreID,
+            directFn: router.getLiveResults,
             model: 'Zenoss.search.FreeSearchModel',
-            root:'results'
+            root: 'results',
+            listeners: {
+                beforeload: {
```

Comment:
listen "beforeload" in "live search" store to handle and show "saved search" store results if user click on combo trigger.